test/Microsoft.AspNet.Mvc.FunctionalTests/CustomUrlHelperTests.cs

```diff
@@ -1,67 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Huh? Where are the tests of `UrlHelperSample.Web`? Everything of interest in that site should still be tested.